server/rosetta/errors.go

```diff
@@ -0,0 +1,153 @@
+// errors.go contains all the errors returned by the adapter implementation
+// plus some extra utilities to parse those errors
```

Comment:
This ends up showing up as package documentation. I'd suggest to move this after the package declaration.

Reply:
prob move to another errors/ directory within rosetta/